waku/v2/protocol/waku_rln_relay/waku_rln_relay_utils.nim

```diff
@@ -560,13 +553,28 @@ when defined(rlnzerokit):
 
     return ok(true)
 
-  proc insertMember*(rlnInstance: ptr RLN, idComm: IDCommitment): bool =
-    var pkBuffer = toBuffer(idComm)
-    let pkBufferPtr = addr pkBuffer
+  proc insertMembers*(rlnInstance: ptr RLN,
+                      index: MembershipIndex,
+                      idComms: seq[IDCommitment]): bool =
+    ## Insert multiple members i.e., identity commitments
+    ## returns true if the insertion is successful
+    ## returns false if any of the insertions fails
```

Comment:
We don't use the `rlnInstance.insertMembers` in rln-relay, there is a wrapper proc `insertMembers` on the `WakuRlnRelay` object, which handles the error. Since zerokit returns only a boolean value, and not an enum with the error variants (as uints), it is only possible to detect the error with a false value.